app/reports/publish_status.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Exports publish information about druids lacking public cocina.
+#   https://github.com/sul-dlss/dor-services-app/issues/5181
+# Input is a CSV of druids generated on purl-fetcher with the query:
+#   Purl.status('public').where.missing(:public_json).pluck(:druid)
+# Invoke via:
+# bin/rails r -e production "PublishStatus.report('druids.csv')"
+class PublishStatus
+  def self.report(druids_file)
+    druids = File.readlines(druids_file).map(&:strip)
```

Comment:
Is the `strip` bit there to remove trailing newlines? If so, you could also consider this hard-to-ferret-out argument to `File::readlines` (because it's defined in `IO::readlines`):

```suggestion
    druids = File.readlines(druids_file, chomp: true)
```

But this is mostly FYI, and I don't expect you to make a change and run the report again.

Reply:
Thanks, wasn't aware of the `chomp` argument, so good to know!